src/MOI_wrapper.jl

```diff
@@ -224,6 +238,25 @@ function load_constraint(
     return
 end
 
+function load_constraint(
+        index::MOI.ConstraintIndex, model::CbcModelFormat,
+        mapping::MOIU.IndexMap, func::MOI.VectorOfVariables,
+        set::MOI.SOS1{Float64})
+    var_indices = [v.value for v in func.variables]
```

Comment:
You should probably use `Int32` here to avoid the vector to be converted afterwards